pkg/vpcmodel/grouping.go

```diff
@@ -187,27 +190,28 @@ func (g *groupedExternalNodes) Name() string {
 
 // given a groupedEndpointsElems returns an equiv item from groupedEndpointsElemsMap if exists,
 // or adds it to groupedEndpointsElemsMap if such an item does not exist
-func (g *GroupConnLines) getGroupedEndpointsElems(grouped groupedEndpointsElems) *groupedEndpointsElems {
+func getGroupedEndpointsElems(grouped groupedEndpointsElems,
```

Comment:
This func should receive a reference to the cache, and use a cache method to check existence of element.
The cache-related functionality should be encapsulated by the cache type.

Reply:
done